src/db.rs

```diff
@@ -844,6 +844,62 @@ impl<T: ThreadMode> DBWithThreadMode<T> {
         Ok(convert_values(values, values_sizes))
     }
 
+    /// Returns `false` if the given key definitely doesn't exist in the database, otherwise returns
+    /// `false`. This function uses default `ReadOptions`.
```

Comment:
These comments suggest that the new methods always return `false`. I assume it should be "otherwise returns `true`"?
It would be nice if the new tests incldued `true` cases to demonstrate.

Reply:
Thanks, I'm going to update the comments. As for the true-case tests: these functions can always return `true` and it will be technically correct. It will require understanding of the RocksdDB internals to write such tests and still it can become broken in a newer version.

Reply:
https://github.com/rust-rocksdb/rust-rocksdb/pull/521

Reply:
Thank you!